MotionMark/resources/runner/motionmark.js

```diff
@@ -402,39 +402,39 @@ class ResultsTable {
     }
 }
 
-window.benchmarkRunnerClient = {
-    iterationCount: 1,
-    options: null,
-    results: null,
-
-    initialize: function(suites, options)
+class BenchmarkRunnerClient {
+    iterationCount = 1;
```

Comment:
I think we should remove this member since it is always 1. We allow running MotionMark multiple times but I think it trued out to cause too much heat and affect the score.

Reply:
I suspect this might be used when running in infrastructure (we apply patches to the benchmark there). Leaving for now.